src/App.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
-import SinglePage from './components/singlePage/singlePage';
+import './App.css';
+import LoginForm from './components/loginForm/loginForm.';
+import './components/layouts/navigation'
+import { BottomNavigation } from '@material-ui/core';
+import Navigation from "./components/layouts/navigation";
 
 function App() {
   return (
-        <SinglePage />
+    <div className="App">
+      // Add your code here
```

Comment:
Please add <SinglePage /> again and throw you Navigation component in there, because <SinglePage /> is the entry point for the Ruby backend!

Reply:
done, bottomNavigation is inserted